plugins/inputs/aliyuncms/README.md

```diff
@@ -133,6 +139,11 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
     ## Or you can specify several dimensions at once:
     # dimensions = '[{"instanceId": "p-example"},{"instanceId": "q-example"}]'
 
+    ## Specified which metric service to capture from Aliyun
+    ##  * cms - Cloud Monitor service (default settings)
+    ##  * rds - Relational Database service
+    # service = 'rds'
+
```

Comment:
Why is there a `service` + `metric_services`? Shouldn't this be removed in favor of the `metric_services` above?

Reply:
The `metric_service` is the general trigger point to initialize the RDS client on a technical level and to activate the function in general. The `service` parameter defines the corresponding `extended` metrics and marks them. These metrics are not available at the standard level (CMS). This means at the API level that you will not get a result for the endpoint if you call it with the wrong metrics. 
It is not marked as an error, but we log it as debug information. If this behavior is ok, I can also remove the `service` parameter and we intentionally call the API endpoint with the wrong information.

Reply:
I'm not sure I understand the scenario at play with these settings then.

Why would I enable "cms" metric_services and not "cms" service? (This naming is a headache) :) Likewise, why would I enable "rds" metric_services and not "rds" service?

Reply:
Hi @powersj, 
> I'm not sure I understand the scenario at play with these settings then.
> Why would I enable "cms" metric_services and not "cms" service? (This naming is a headache) :) Likewise, why would I enable "rds" metric_services and not "rds" service?

The corresponding parameters are linked to each other.

The `metric_service` parameter is the generic option and generally activates the option. It initiates the separate client that processes the request to the dedicated API endpoint.

The `service` parameter at the metric level is the trigger point for calling the API. In the current implementation, it is the RDS service of Alicloud and extracts the separately specified values.

If we don't have a switch for the metric level, we are intentionally calling the RDS API in the wrong context to get metrics e.g. we expect CMS default metrics, thus burdening the Alicloud rate limit.

> You mentioned extended metrics. We are not collecting these today for cms right?
> Should this be a follow-on PR and this PR instead be focused only on adding the new RDS service?

The CMS service and the dedicated APIs of the services themselves are not compatible and are separate endpoints. When I talk about extended metrics, I'm talking about the dedicated RDS API endpoint that this PR is already using.

Reply:
Hi @powersj, is there any ambiguity regarding the functionality or can I proceed to write new tests?